sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -36,7 +41,16 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   <UsingTask TaskName="ZipDeployTask"
             AssemblyFile="$(_FunctionsTaskAssemblyFullPath)"/>
 
-  <Target Name="_FunctionsPostBuild" AfterTargets="Build">
+    <Target Name="_FunctionsPreBuild" BeforeTargets="Build">
```

Comment:
It looks like the new line is indented differently than the old line; is it what you expect in the real file?

Reply:
nope that was an unexpected indentation from working in different editors. Have fixed this.